axelor-stock/src/main/java/com/axelor/apps/stock/web/TrackingNumberConfigurationController.java

```diff
@@ -0,0 +1,24 @@
+package com.axelor.apps.stock.web;
+
+import com.axelor.apps.stock.db.TrackingNumberConfiguration;
+import com.axelor.apps.stock.service.TrackingNumberConfiguratorService;
+import com.axelor.inject.Beans;
+import com.axelor.rpc.ActionRequest;
+import com.axelor.rpc.ActionResponse;
+import com.google.inject.Singleton;
+
+@Singleton
+public class TrackingNumberConfigurationController {
+
+  public void checkSequenceAndBarcodeTypeConfigConsistency(
+      ActionRequest request, ActionResponse response) {
+    try {
+      TrackingNumberConfiguration config =
+          request.getContext().asType(TrackingNumberConfiguration.class);
+      Beans.get(TrackingNumberConfiguratorService.class)
+          .checkSequenceAndBarcodeTypeConfigConsistency(config);
+    } catch (Exception e) {
+      response.setError(e.getLocalizedMessage());
```

Comment:
Please use `TraceBackService.trace(response, e, ResponseMessageType.ERROR);` to trace the full exception and to return an error with the message.

Reply:
This method is used like an action-validate, we don't really need to trace the error, we just want the user to have a error message if needed. This methods relies on consistency check between sequence and barcode type process which only throw exception when something wrong happend. Catching it and restore it to the user and block following actions is the behaviour that we want here. Also adding traceback after response.setError will display in double the error message to user.